geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportParams.scala

```diff
@@ -20,7 +20,7 @@ trait FileExportParams {
   @Parameter(names = Array("--gzip"), description = "Level of gzip compression to apply to output, from 1-9")
   var gzip: Integer = null
 
-  @Parameter(names = Array("-F", "--output-Format"), description = "File format of output files (csv|tsv|gml|json|shp|avro)")
+  @Parameter(names = Array("-F", "--output-format"), description = "File format of output files (csv|tsv|gml|json|shp|avro)")
```

Comment:
We're going to have to be careful with changes like this in the future. The autocomplete won't update so if they already have an autocomplete for that command installed and we change the param, tab complete will provide and invalid name.